core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -248,14 +248,15 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         onlyPreExpiration()
         fees()
     {
+        require(_checkCollateralization(collateralAmount, numTokens));
```

Comment:
With how you've designed the collateralization checks below, it seems pretty easy to add an `||` check for the new tokens and the total position. Curious to hear your thoughts now that we know more about how the implementation will look. Are there any disadvantages to the `||` check?

Reply:
Yep, the implementation would be as simple as to move this line down below line 261 and do:

```
require(_checkCollateralization(collateralAmount, numTokens) || _checkCollateralization(positionData));
```

I don't have a strong opinion either way. Seems simpler to just not allow it, one less case that we have to prove is correct. (E.g., the GCR will now decrease after creating tokens).

Reply:
Happy to make this change in a follow up if we think that's preferable.